client/src/views/mobile/TournamentMobileView.tsx

```diff
@@ -128,10 +141,10 @@ const TournamentMobileView = ({
                           mr: 1,
                         }}
                         >
-                          <BruteRender brute={fight.brute1} />
+                          {brute1 && <BruteRender brute={brute1 as BruteForRender} />}
```

Comment:
It's better to avoid using `as` in TS, as it removes the type safety. If your brute1 object has all the fields necessary for <BruteRender />, it shouldn't be needed